app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# Article Now is a Local Variable %>
+<%= form_with model: article do |form| %>
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+    <% article.errors.full_messages_for(:title).each do |message| %>
+      <div><%= message %></div>
+    <% end %>
+  </div>
+
+  <div>
+    <%= form.label :body %><br>
+    <%= form.text_area :body %><br>
+    <% article.errors.full_messages_for(:body).each do |message| %>
+      <div><%= message %></div>
+    <% end %>
+  </div>
+
+  <div>
+    <%= form.label :status %><br>
+    <%= form.select :status, ['public', 'private', 'archived'], selected: 'public' %>
```

Comment:
The same Ruby rules apply to ERB files. You can use `%w` notation for declaring an array of strings.

Reply:
Got it! %w for an array of strings and %i for an array of symbols.